health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/BoundaryRelationCreate.js

```
@@ -574,104 +574,109 @@ const onConfirmClick=()=>{
-                    {creatingData&&<LoaderWithGap text={t("DATA_SYNC_WITH_SERVER")} />}
-                </React.Fragment>
-
-
+              <React.Fragment>
+                {firstPage && (
+                  <div>
+                    <Card type={"primary"} variant={"viewcard"} className={"example-view-card"}>
+                      <div style={{ display: "flex", justifyContent: "space-between" }}>
+                        <div className="hierarchy-boundary-heading">{t("CREATE_BOUNDARY_HIERARCHY")}</div>
+                      </div>
+                      <div style={{ height: "1.5rem" }}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)